packages/jaeger-ui/src/components/Monitor/ServicesView/index.test.js

```diff
@@ -361,6 +361,7 @@ describe('mapStateToProps()', () => {
       metrics: originInitialState,
       services: [],
       selectedService: 's1',
+      selectedSpanKind: 'server',
```

Comment:
do we have behavioral tests? It would be good to have tests simulating a change in span kind

Reply:
Yup, I could do better with test coverage; added behavioural tests in: [62a71da](https://github.com/jaegertracing/jaeger-ui/pull/2012/commits/62a71dae01304c159731e6f54661db13c1dfeff9).